.github/workflows/app-build.yaml

```diff
@@ -30,9 +30,9 @@ jobs:
         run: ./gradlew --no-daemon --info assembleDebug
       - name: Split APK release types
         run: |
-          mkdir -p build/jellyfin-publish/release-libre build/jellyfin-publish/release-proprietary;
-          mv app/build/outputs/apk/*/*/jellyfin-android-*-libre-debug.apk build/jellyfin-publish/release-libre/;
-          mv app/build/outputs/apk/*/*/jellyfin-android-*-proprietary-debug.apk build/jellyfin-publish/release-proprietary/;
+          mkdir -p build/jellyfin-publish;
+          mv app/build/outputs/apk/*/*/jellyfin-android-*-libre-debug.apk build/jellyfin-publish/;
+          mv app/build/outputs/apk/*/*/jellyfin-android-*-proprietary-debug.apk build/jellyfin-publish/;
```

Comment:
```suggestion
          mkdir -p build/jellyfin-publish
          mv app/build/outputs/apk/*/*/jellyfin-android-*-libre-debug.apk build/jellyfin-publish/
          mv app/build/outputs/apk/*/*/jellyfin-android-*-proprietary-debug.apk build/jellyfin-publish/
```

Reply:
they should not be nescessary, but they don't hurt either?

Reply:
They aren't a real problem, no. They just trigger my OCD cause Java :rofl: